cli/src/print.rs

```diff
@@ -38,39 +38,42 @@ impl<W: Write> Print<W> {
             Payload::Update(n) => affected(*n, "updated")?,
             Payload::ShowVariable(PayloadVariable::Version(v)) => self.write(format!("v{v}"))?,
             Payload::ShowVariable(PayloadVariable::Tables(names)) => {
-                let mut table = get_table(["tables"]);
+                let mut table = get_table(&["tables"]);
                 for name in names {
-                    table.add_row([name]);
+                    table.add_record([name]);
                 }
-
+                let table = build_table(table);
                 self.write(table)?;
             }
             Payload::ShowColumns(columns) => {
-                let mut table = get_table(vec!["Field", "Type"]);
+                let mut table = get_table(&["Field", "Type"]);
                 for (field, field_type) in columns {
-                    table.add_row([field, &field_type.to_string()]);
+                    table.add_record([field, &field_type.to_string()]);
                 }
-
+                let table = build_table(table);
                 self.write(table)?;
             }
             Payload::ShowIndexes(indexes) => {
-                let mut table = get_table(vec!["Index Name", "Order", "Description"]);
+                let mut table = get_table(&["Index Name", "Order", "Description"]);
                 for index in indexes {
-                    table.add_row([
+                    table.add_record([
                         index.name.to_string(),
                         index.order.to_string(),
                         index.expr.to_sql(),
                     ]);
                 }
+                let table = build_table(table);
                 self.write(table)?;
             }
             Payload::Select { labels, rows } => {
+                let labels = &labels.iter().map(AsRef::as_ref).collect::<Vec<&str>>();
```

Comment:
This could be good hint for us.
https://github.com/zhiburt/tabled/blob/308cdca66ad2dac07f83ab3eba9e289819d5782c/src/builder.rs#L193-L197
```rust
    pub fn set_columns<H, T>(&mut self, columns: H) -> &mut Self
    where
        H: IntoIterator<Item = T>,
        T: Into<Cow<'a, str>>,
    {
```

For our case, `Into<Cow<'a, str>>` would be unnecessary so
```rust
fn get_table<'a, T: IntoIterator<Item = &'a str>>(header: T) -> Builder {
    let mut table = Builder::default();
    table.set_columns(header);

    table
}
```

Then
```rust
let labels = labels.iter().map(AsRef::as_ref);
```
and we also don't need to add borrow for other header labels
e.g.
```rust
// prev
let mut table = get_table(&["Index Name", "Order", "Description"])

// now
let mut table = get_table(["Index Name", "Order", "Description"])
```